src/Build/Evaluation/Evaluator.cs

```diff
@@ -2076,6 +2102,17 @@ private void EvaluateImportElement(string directoryOfImportingFile, ProjectImpor
                 PerformDepthFirstPass(importedProjectRootElement);
             }
 
+            if (!importedAnything && hasCondition)
```

Comment:
Should there be an else branch here? If there is no condition and nothing was imported, maybe it was a glob import and the glob expanded to nothing. Or the import expression referenced properties which resolved to empty.

@jeffkl Sounds about right? Are there other cases where an import with no condition evaluates to empty?